tdigest/tdigest.proto

```diff
@@ -15,10 +15,10 @@ message MergingDigestData {
     double min = 3;
     double max = 4;
     double reciprocalSum = 5;
+    bool debug = 6;
 }
 
 message Centroid {
     double mean = 1;
     double weight = 2;
-    repeated double samples = 3;
```

Comment:
I'd have to brush up on my protobuf… can you safely remove stuff? :)

Reply:
You can if you have no expectations on the two being compatible. This is a fork specifically for the smart gateway. I don't plan on submitting this back.

Reply:
Ah, ok! I didn't realize that. :)

Reply:
Unrelated, we could pluck this out and make a standalone lib. ;)

Reply:
(or they could)

Reply:
Probably should even. This being one tiny portion of the veneur project.